File: android/filtering-content/GetProductWithinRating.java
// DocSection: filtering_get_product_within_rating
// Gets items whose rating is at least 6.5 and at most 9
CompletionStage<ContentItemsListingResponse> items = client.getItems(
    DeliveryParameterBuilder.params().filterRange("elements.product_rating", "6.5", "9").build()
);
// EndDocSection

File: ios/filtering-content/GetProductWithinRating.swift
// DocSection: filtering_get_product_within_rating
// Gets products whose rating is at least 7 but no greater than 9
let customQuery = "items?elements.product_rating[range]=7,9"

client.getItems(modelType: Article.self, customQuery: customQuery) { (isSuccess, itemsResponse, error) in
    if isSuccess {
        if let articles = itemsResponse?.items {
            // Use your items here
        }
    } else {
        if let error = error {
            print(error)
        }
    }
}
// EndDocSection
